app/src/main/res/values/strings.xml

```diff
@@ -149,4 +149,8 @@ We engage our community by contributing to open source, collaborating with the g
     <string name="url_terms">https://anitab.org/terms-of-use/</string>
     <string name="url_privacy">https://anitab.org/privacy-policy/</string>
     <string name="url_code_of_conduct">https://ghc.anitab.org/code-of-conduct/</string>
+    <string name="task_update_confirm">Update task</string>
+    <string name="task_update_title">Complete task?</string>
+    <string name="task_update_mesg">Are you sure you want to complete the task: </string>
+    <string name="task_update_mesg2">\?</string>
```

Comment:
Change this also, to "task_update_message2"

Reply:
done!